### DISCUSSION TITLE
I messed my XCode project files

### DISCUSSION THREAD

**Discussion by JansonJS** [sst-inc/help#17] [Q&A - iOS] [answered]
My Xcode resources files messed up and now the project doesn't work, please help

**Comment by qin-guan** [comment]
Please provide more details such as:

- What errors you're getting
- What files you deleted

Attaching your project folder helps too.

**Reply by jiachenyee** [reply]
You might want to read [this](https://github.com/sst-inc/help/blob/main/README.md) to ask better questions that will help us better understand the issue.

**Comment by qin-guan** [comment, marked as answer]
You can also try recreating the app

**Reply by JansonJS** [reply]
Is there anyway to import all the project items into the new one?

**Reply by qin-guan** [reply]
To reduce issues, we recommend you recreate it
